style.css

```diff
@@ -0,0 +1,1280 @@
+  }
+
+  .social__item-vk {
+    width: 78%;
+  }
+
+  .social__item-fb {
+    width: 78%;
+  }
+
+  .footer__scroll {
+    width: 23px;
+    height: 30px;
+  }
+}
+
+@media (max-width: 992px) { 
+  .first-page {
+    background: linear-gradient(to right, #000 318px, #fff 318px);
+    min-height: 620px;
+  }
+  
+  .header {
+    height: 63px;
+    margin-bottom: 54px;
+  }
+
+  .header__logo {
+    padding: 25px 0 0 36px;
+  }
+
+  .header__logo-loco {
+    font-size: 20px;
+    margin-right: 165px;
+    letter-spacing: 1.8px;
+    font-weight: 600;
+  }
+
+  .header__lang {
+    padding-top: 1px;
+  }
+
+  .header__lang-ru {
+    font-size: 13px;
+    letter-spacing: .5px;
+  }
+
+  .lang__vector {
+    width: 9px;
+  }
+
+  .header__menu {
+    height: 63px;
+    width: 63px;
+  }
+
+  .header__menu-line {
+    width: 24px;
+    height: 3px;
+    margin: 4px 19px;
+  }
+
+  .header__menu-icon {
+    height: 18px;
+  }
+
+  .wrap {
+    margin-bottom: 21px;
+  }
+
+  .wrap__main-img {
+    height: 357px;
+    margin: 0 36px 49px 0;
+  }
+
+  .wrap__img {
+    padding-left: 128px;
+  }
+
+  .wrap__heading {
+    font-size: 55px;
+    line-height: 57px;
+    margin-bottom: 13px;
+  }
+
+  .wrap__article {
+    line-height: 15px;
+    width: 315px;
+    letter-spacing: 0;
+    margin-bottom: 27px;
+    font-size: 10px;
+  }
+
+  .wrap__button-next {
+    height: 30px;
+    padding: 0 26px;
+    line-height: 18px;
+    font-size: 10px;
+    margin-right: 39px;
+    letter-spacing: -.9px;
+  }
+
+  .wrap__button-more {
+    font-size: 10px;
+    letter-spacing: -1.2px;
+  }
+
+  .button-more__vector {
+    padding-left: 11px;
+    width: 31px;
+  }
+
+  .footer__social-networks {
+    padding-left: 36px;
+  }
+
+  .footer__social-button {
+    width: 38px;
+  }
+
+  .footer__scroll {
+    width: 21px;
+    height: 26px;
+  }
+}
+
+@media (max-width: 880px) {
+  .wrap__img,
+  .wrap__main-img,
+  .wrap__article,
+  .wrap__button-next,
+  .wrap__button-more,
+  .button-more__vector,
+  .header__lang,
+  .footer__social-networks {
+    display: none;
+  }
+
+  .first-page {
+    background: #fff;
+    min-height: auto;
+  }
+
+  .header {
+    background: #fff;
+    height: 146px;
+    margin: 0;
+  }
+
+  .header__logo {
+    padding-left: 62px;
+    padding-top: 17px;
+  }
+
+  .header__logo-loco {
+    font: 800 69px 'Montserrat-Regular', sans-serif;
+    color: #000;
+    letter-spacing: 6px;
+    margin: 0;
+  }
+
+  .header__menu {
+    width: 163px;
+    height: 163px;
+  }
+
+  .header__menu-icon {
+    height: 44px;
+  }
+
+  .header__menu-line {
+    width: 61px;
+    height: 7px;
+    margin: 10px 50px;
+  }
+
+  .wrap__content {
+    display: block;
+    background-image: url(./Img/mainpic.png);
+    background-repeat: no-repeat;
+    width: 100%;
+    height: 878px;
+    background-size: cover;
+    margin: 0;
+  }
+
+  .wrap__heading {
+    padding: 0 18%;
+    font: 700 93px/93px 'Montserrat-Bold', sans-serif;
+    color: #000000;
```

Comment:
Codacy found an issue: [Expected "#000000" to be "#000" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/Flack_project/pullRequest?prid=5633354)